### PR TITLE
Episode 4

### PR DESCRIPTION
**Pull request by Hacklor**
@matthijsgroen Please review.

The text for the episode is finished, but I still need to implement the other 3 operations from the final exercise myself. I will do this on tuesday.

### CODE REVIEW

File: episode4/README.md [hunk 1]
```diff
@@ -1,64 +1,400 @@
...
+```js
+describe("Regression test", () => {
+	it("can combine multiple chained commands", () => {
+		calculator
+			.add(7)
+			.subtract(2)
+			.square()
+			.multiply(4)
+
+		expect(calculator.result).to.eql(100)
```
Comment (Hacklor) [inline episode4/README.md]: `7 - 2 ^ 2 * 4 = -9`

Pure technically speaking. But we haven't covered the lazy part of coming to the result yet. I'm gonna write a note, cus it bothers me. :P

File: episode3/README.md [hunk 2]
```diff
@@ -20,13 +20,17 @@ describe("Regression tests", () => {
...
-functions. They have a slightly different way how the scope works, but
-we get to that later 😅
 
-In short, we will refactor this:
+What is [refactoring][refactoring]?
+The process of refactoring is to structure software by applying a series of refactorings without changing its observable behaviour.
+The last part is important, without changing its observable behaviour. Behaviour can be observed
+with tests and this way we can make sure it does not change while we are structuring our code.
 
+Now we know the code works, we could refactor the `functions` into a version with [lamda][lamdas],
```
Comment (matthijsgroen) [inline episode3/README.md]: the link should be `[lamdas][lamda]`

Comment (Hacklor) [reply episode3/README.md]: Technically, both are incorrect. It should be `lambdas`. Changing it.

**Comment by Hacklor** [comment]
Changed episode 3 and 4 based on your comments. Please push the button!

**Pull request merged**